packages/@aws-cdk/aws-codestarnotifications/test/notification-rule.test.ts

```diff
@@ -0,0 +1,196 @@
+import '@aws-cdk/assert-internal/jest';
+import * as cdk from '@aws-cdk/core';
+import * as notifications from '../lib';
+import {
+  FakeCodeBuild,
+  FakeCodePipeline,
+  FakeSlackTarget,
+  FakeSnsTopicTarget,
+} from './helpers';
+
+describe('Rule', () => {
+  let stack: cdk.Stack;
+
+  beforeEach(() => {
+    stack = new cdk.Stack();
+  });
+
+  test('created new notification rule with source', () => {
+    const project = new FakeCodeBuild();
+
+    new notifications.NotificationRule(stack, 'MyNotificationRule', {
+      source: project,
+    });
+
+    expect(stack).toHaveResourceLike('AWS::CodeStarNotifications::NotificationRule', {
+      Resource: 'arn:aws:codebuild::1234567890:project/MyCodebuildProject',
```

Comment:
This is a really weird test, considering this value is nowhere to be found in the test, it's just a helper constant... maybe extract this ARN to a value you can reference here? Or maybe a static field of `FakeCodeBuild`?

Reply:
Ok, changed to static field of `FakeCodeBuild`

Reply:
Already adjusted.